### ISSUE TITLE
Is this Abandoned?

### ISSUE THREAD

**Issue by adamwbb** [ACCORD5/TrellisDesk#92]
is this abandoned?
I need the email support protocols that this version provides but it seems it hasnt been updated in over a year. plus since its alpha means tis full of bugs and very unstable for production environment.

**Comment by DanielRuf** [comment]
@adamwbb 

Yes, this project is completely abandoned. I did many changes on the code as KAOSFORGE and DanielRuf and as far as I know the alpha version is not a complete rewrite but introduced many new features, improvements, user interface, ... might work. 

But I saw many organizations using the old stable 1.0.4 (or newer) version, but it might not be good to trust this old version, there might be many security problems, esspecially security holes.

I strongly suggest looking for another (opensource) solution.

**Comment by marcmoris** [comment]
Exists there any limitations, if my company takes in charge the support and the development of this software?

**Comment by DanielRuf** [comment]
Sure, forks are welcome and there is a license which you should read.

https://github.com/ACCORD5/TrellisDesk/blob/master/LICENSE.txt

**Comment by marcmoris** [comment]
Super, thank you!

We create a new website : http://trellisdesk.ca

**Comment by DanielRuf** [comment]
But you should also ask the original developers regarding domain name, product name and so on.

I can not give you legal advices as I am not one of the original owners but also worked on the code for a long time.

But currently this project seems to be dead and the developers work now on other things.

**Comment by djtarazona** [comment]
Thanks — we are in communication via email.  Let me run this by the team and I'll get back to you.

This project is no longer in active development.  Indeed, our team has moved on to other projects.  You are more than welcome to fork the repository and continue development.

**Comment by marcpires** [comment]
Needing help count with my. One of my clients use it and a very dependable. So, i decided to continue to supporting the app.

**Comment by marcpires** [comment]
@djtarazona if the project is no longer in active development, why not transfer the repo to community instead making forks ?

Hope hear from you soon

**Comment by djtarazona** [comment]
@marcpiresrj what do you suggest? Transferring the repo out of this organization, or inviting new members to this organization?

**Comment by marcpires** [comment]
@djtarazona I think we could come up with something like the TrellisDesk foundation  - if it's to hard to do, and invite whoever wants to contribute to be project.

As I said, i have clients that use the Trellis Desk and would be glad  to help with its development. Let's hear from other people here and see what they suggest.

Cheers.

**Comment by DanielRuf** [comment]
There are so many (better) alternative like OTRS which are also more secure and have more users.

When I started contribution (as KAOSFORGE and DanielRuf) on the next version, there were many things to improve, many ideas and issues.

A foundation might be too much work and makes no sense here, in my opinion.

Normally you create a fork on GitHub and work on it and make the pull requests.

I would also use PDO instead of the deprecated MySQL methods.

@djtarazona if there are good contributors who make good pull requests and do this often should be added to the organization.

Transferring this repo out of the organization makes no sense to me. There is already a fork with the organization TrellisDesk https://github.com/TrellisDesk

**Comment by marcpires** [comment]
Hi @DanielRuf, I see. I'll send my contribution to this fork then. Thanks.

**Comment by djtarazona** [comment]
I'm happy to add dedicated / good contributors to the organization so that they have permission to write to the repo and manage pull requests.

**Comment by Blopes5** [comment]
Oh... looking for an answer to my issue I discovered the project it's abandoned... how sad
I've seen about OTRS, but it only runs on their server, and its paid.
I'm looking for a free help desk manager, could you indicate me one?

**Comment by DanielRuf** [comment]
@Blopes5 OTRS has a free community version that you can install  on your own server.

https://www.otrs.com/download-open-source-help-desk-software-otrs-free/
